packages/client/test/integration/PublisherKeyExchange.test.ts

```diff
@@ -42,15 +43,15 @@ describe('PublisherKeyExchange', () => {
         return stream
     }
 
-    const createGroupKeyRequest = (
+    const createGroupKeyRequest = async (
```

Comment:
Future refactoring idea: extract a function `createGroupKeyRequest` in source code so test can use that instead?

Reply:
Alternatively we could use a real `SubscriberKeyExchange` in that test